### DISCUSSION TITLE
setting up opc ua moniotring for the first time , can i use this to read 60k tags at once every minute

### DISCUSSION THREAD

**Discussion by visarya** [gopcua/opcua#412] [Q&A] [answered]
I working on ua client for one of the projects. i recently stared with golang and wanted to be sure the solution will work for ua.

**Comment by magiconair** [comment, marked as answer]
I think 60k nodes in a single read might be pushing it since the response most likely exceeds the maximum response size. I think that this will probably also put some burden on your UA server. I would suggest you spread this out over the minute. Even 1000 tags per second might be a lot. The library can handle it. I'm just not sure if the protocol or the UA server can.

**Reply by visarya** [reply]
thanks for the reply, I am able to use opc da to fetch more than 100k tags in less than 30 seconds using different lib in go. Since ua is on top of same underline opc, i think the server should be able to handle it.

**Reply by magiconair** [reply]
Which library are you currently using?

**Reply by magiconair** [reply]
The easiest thing is to just try it.
